zwave_js_server/protocol/__init__.py

```diff
@@ -0,0 +1,25 @@
+"""Provide a protocol for zwave-js-server."""
+from typing import Any, Callable, Protocol
+
+
+class ModelType(Protocol):
+    """Represent a model event type."""
+
+    type: Any
+    data: Any
+
+
+class ProtocolType(Protocol):
+    """Represent a protocol module type."""
+
+    Type: Any
+    Handler: Any
+
+
+def get_handler(protocol: ProtocolType, data: ModelType) -> Callable:
```

Comment:
This could be just on a protocol instance.

Reply:
I've moved this into the EventBase as a helper.